extension/content/components/recipes/form/filters/SamplingOptions.js

```diff
@@ -249,7 +250,8 @@ function getFilterObjectFromData() {
 
   let filterObject;
   if (data.filter_object) {
-    filterObject = data.filter_object.find((fo) =>
+    const [filterObjects, _omitAdditionalFO] = partitionFO(data.filter_object);
```

Comment:
fyi: you dont need the `_omitAdditionalFO`. since you are not trying to access an item in the array with a higher index you can simply omit it like:
```js
const [filterObjects] = partitionFO(data.filter_object);
```